src/App.tsx

```diff
@@ -15,48 +16,136 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortBy {
+  Alfabet = 'alphabet',
+  Length = 'length',
+  Indexdesc = 'reverse Default',
+  Default = 'Default',
+}
+
+const sortAlfabet = (a: string, b: string, direction: boolean) => {
```

Comment:
There is a typo here. 
```suggestion
const sortAlphabet = (a: string, b: string, direction: boolean) => {
```